client/graphviz.js

```diff
@@ -15,11 +15,28 @@
   async function makedot($item, item) {
     const asSlug = (name) => name.replace(/\s/g, '-').replace(/[^A-Za-z0-9-]/g, '').toLowerCase()
 
-    let text = item.text
-    if (text.match(/^DOT MEHAFFY$/)) {
-      return diagram ($item, item)
-    } else if (m = text.match(/^DOT ((strict )?(di)?graph)\n/)) {
+    var text = item.text
+    if (m = text.match(/^DOT FROM ([a-z0-9-]+)($|\n)/)) {
+      let site = $item.parents('.page').data('site')||location.host
+      let slug = m[1]
+      var page = null
+      try {
+        page = await wiki.site(site).get(`${slug}.json`, (err, page) => page)
+      } catch (err) {}
```

Comment:
does this want to log the error to the console?
```suggestion
      } catch (err) {console.error('failed redirect', site, slug, err)}
```